mqtt/mqttd/src/main.rs

```diff
@@ -1,7 +1,6 @@
 use std::{convert::TryInto, env, io};
```

Comment:
Just a general note, we should probably keep an eye on the length of `main`. I don't want what we have in `iotedged` to happen again where the `main` method is huge and not tested. Nothing required to change in this PR. Edits to `main` reminded me.